starters/vue3-xstate-css/README.md

```diff
@@ -0,0 +1,196 @@
+# Vue 3, XState, and CSS Starter Kit
+
+This starter kit uses [Vue 3](https://vuejs.org/), [XState](https://xstate.js.org/) for state management, and pure CSS for styling.
+
+**How to use this starter kit**
+
+```sh
+npm create @this-dot/starter -- --kit vue3-xstate-css
+```
+
+<sub>(use any of `npm` / `yarn` / `pnpm`)</sub>
+
+For more setup options, check out our [setup instructions in the wiki](https://github.com/thisdot/starter.dev/wiki/Setup-Instructions).
+
+## Tech Stack
+
+- [Vue 3](https://vuejs.org/)
+- [XState](https://xstate.js.org/)
+- CSS
+
+This kit is also set up to show the XState visualizer when run locally, to help you see what your state machines look like and how they work.
+
+### Included Tooling
+
+- [Vite](https://vitejs.dev/) - build / bundle tool
+- [Vue Router](https://router.vuejs.org/) - navigation
+- [Cypress](https://docs.cypress.io/guides/component-testing/overview) - component / unit testing
+- [Storybook](https://storybook.js.org/) - documents component designs
+- [ESLint](https://eslint.org/) - code consistency and best practices
+- [Prettier](https://prettier.io/) - code formatting
+
+**Type Support for `.vue` Imports in TS**
+
+This project is setup to handle TypeScript files. However, TypeScript cannot handle type information for `.vue` imports by default, so we replace the `tsc` CLI with `vue-tsc` for type checking. In editors, we need [TypeScript Vue Plugin (Volar)](https://marketplace.visualstudio.com/items?itemName=Vue.vscode-typescript-vue-plugin) to make the TypeScript language service aware of `.vue` types.
+
+## Getting Started
+
+### Prerequisites
+
+- [Node.js 16.8](https://nodejs.org/) or later installed
+
+### Development
+
+Once you've generated your new starter kit following the instructions above:
+
+- `cd` into your project directory and run `npm install` to install the dependencies.
+- Run `npm run dev` to start the development server.
+- Click the [http://localhost:5173] link from within your terminal. This will open that link in a tab to view the application, and also launch the XState inspector for you in a second tab.
```

Comment:
small markdown issue here

<img width="362" alt="Screenshot 2023-03-31 at 4 46 04 PM" src="https://user-images.githubusercontent.com/67210629/229251189-3ceeb8bb-29da-4b2f-b776-a8796b9efda0.png">

```suggestion
- Click the [http://localhost:5173](http://localhost:5173) link from within your terminal. This will open that link in a tab to view the application, and also launch the XState inspector for you in a second tab.
```

Reply:
Dang, the direct link with <> was causing crashes but I thought this would work without specific text. Good catch!